proto/common.proto

```diff
@@ -75,6 +75,20 @@ message WorkerNode {
   // It's populated by meta node, when the worker node is added by meta node.
   // It's not persistent in meta store.
   optional uint64 started_at = 9;
+
+  // Meta may assign labels to worker nodes to partition workload by label.
+  // This is used for serverless backfilling of materialized views.
+  // If a WorkerNode has labels set it indicates that this node only accepts certain work.
+  // If a WorkerNode has no labels (empty map or null value) the node accepts any work.
+  // The materialized view has to have both a matching key and value to be scheduled on a node.
+  // E.g. {"some":"value"} can be scheduled on a node with the labels
+  // { 
+  //   "other":"thing",
+  //   "some":"value"
+  // }
+  // but {"some":"thing"} can not be scheduled on that node.
+  // Worker nodes with an empty node_labels map MVs with an empty or null node_label map.
+  map<string, string> node_labels = 10;
```

Comment:
Actually I don't really get the purpose to making the labels a `map` here. Besides scheduling, I can't think of any other scenarios that require the use of `map` for differentiation. Is this over design?

Reply:
I am also fine with just using a `string`. IMHO the only advantages of using a map is 
- being close to the K8s way of doing things 
- having a more general purpose interface
CC @shanicky

Reply:
Also, what do we want to use to indicate that a streaming job is unschedulable. IMHO we could just use this constant. Are you ok with that? 
```
node_labels = { "workload": "unschedulable" }
```

If we do not want to agree on a constant, then I'd suggest that the controller looks up the `node_labels` of all the nodes in the cluster and considers a streaming job stuck if there is no matching `node_label`